src/Symfony/Bundle/SecurityBundle/CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 CHANGELOG
 =========
 
+3.3.0
+-----
+
+ * Deprecated registering by convention the `UserPasswordEncoderCommand` command 
```

Comment:
Is this deprecation necessary? The command will be registered by the SecurityBundle anyway, won't it?